webapp/tests/test_models.py

```diff
@@ -0,0 +1,109 @@
+# Core
+import unittest
+
+# Modules
+import responses
+
+# Local
+from webapp import models
+from webapp.tests.fixtures.snapcraft_forum_responses import (
+    documentation_outline_topic,
+    frontpage_by_id,
+    getting_started_wiki,
+    getting_started_by_id,
+    configuration_options_redirect,
+)
+
+
+class TestDiscourseDocs(unittest.TestCase):
+    def setUp(self):
+        responses.add(
+            documentation_outline_topic,
+            frontpage_by_id,
+            getting_started_wiki,
+            getting_started_by_id,
+            configuration_options_redirect,
+        )
+
+        self.discourse = models.DiscourseDocs(
+            base_url="https://forum.snapcraft.io", frontpage_id=3781
+        )
+
+    def test_get_document(self):
```

Comment:
I really don't think it's a good idea to have tests for a function and have every case in the same test.

You should have a function with a name that explains what are you testing, this makes it clearer when you make a change. It's much more understanble what broke exaclty, and not only `test_get_document`